features/customers/search_with_autocomplete.feature

```diff
@@ -34,7 +35,8 @@ Scenario:search with other information
     | Organ      | Milk      | dancingfox@mail.com | 100 bway      |  SAF | CA    |
 
   When I fill "search_field" autocomplete field with "Fox"
-  Then I should see autocomplete choice "Armando Fox"
+  Then I should see autocomplete choice "show all"
```

Comment:
It is not tested that when pressing "show all", we would go to the page of everything.

Reply:
You're right, I missed this test case.